samples/net/http_client/README.rst

```diff
@@ -69,12 +74,9 @@ port is 8000.
 HTTP Server
 ===========
 
-A very simple HTTP server is provided in net-tool project.
-
-The net-tools can be downloaded from
-
-    https://github.com/zephyrproject-rtos/net-tools
-
+Sample code for a very simple HTTP server can be downloaded from the
```

Comment:
There is no conflict as PR #409 is modifying only https_client directory (note the **s** in the name).